### ISSUE TITLE
LaTeX compilation error: LaTeX Error: File `standalone.cls' not found. 

### ISSUE THREAD

**Issue by Lay-Yin** [rstudio/tinytex#441]
I've installed TinyTeX by chocolatey `choco install manim-latex.` It is a dedicated package for Manim based on TinyTeX which contains all the required packages that Manim interacts with.
After installation, my code couldn't run because of " You do not have package standalone.cls installed."
![image](https://github.com/rstudio/tinytex/assets/131653407/2295a7a6-88a5-47bb-bdfb-8eb0922a3a29)
Any possible solution will be welcome and appreciated！

**Comment by cderv** [comment]
This TinyTeX package of chocolatey: https://community.chocolatey.org/packages/tinytex

it is built based on `TinyTeX-1` bundle as mentioned in README: https://github.com/rstudio/tinytex-releases#chocolatey

This bundle only includes some package in the installation as described in https://github.com/rstudio/tinytex-releases#releases

`standalone` package is available in TinyTeX bundle which is bigger size. This bundle is not available in chocolatey. 

This means the `standalone` CTAN package needs to be installed manually, after installation. 

> It is a dedicated package for Manim based on TinyTeX which contains all the required packages that Manim interacts with.

If `manim-latex` requires it, then it needs to install this package probably in post installation step of its choco package. 


Basically, this is a `manim-latex` issue, and you should report to in its repo: https://github.com/ManimCommunity/manim
